script/github_releases.py

```diff
@@ -1,61 +1,125 @@
 #!/usr/bin/env python
+import logging
+from pprint import pprint
+
+import requests
+import argparse
+import re
 
 from sys import exit, stderr
 from os import getenv, path
-from re import split, IGNORECASE
-from requests import get, patch
 
+log = logging.getLogger(__name__)
+
+RE_LOG_HEADER = re.compile(r"## streamlink\s+(\d+\.\d+\.\d+(?:-\S+)?)\s+\(\d{4}-\d{2}-\d{2}\)\n+", flags=re.IGNORECASE)
+RE_GITLOG = re.compile(r"(.*?)(```\n.*?\n```)", re.DOTALL)
+
+template = """
```

Comment:
Would it make sense moving the template to its own file? It would look a bit cleaner then.